### ISSUE TITLE
uBlock Origin conflict on ytInitialPlayerResponse

### ISSUE THREAD

**Issue by FlawTECH** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#14]
Hi, there seems to be a conflict with uBlock Origin when trying to re-define the ytInitialPlayerResponse property. Here's what I get with both the script and the extension enabled :
![image](https://user-images.githubusercontent.com/36300135/119674479-84303880-be3c-11eb-96eb-82adca73319e.png)

The script works fine without uBlock Origin though, so that's my "bypass" in the meantime.

Thank you

**Comment by FlawTECH** [comment]
After doing some research, it looks like this is due to uBlock Origin setting ytInitialPlayerResponse through one of the filters with configurable: false before this script is even running, so it cannot intercept the call to defineProperty

**Comment by zerodytrash** [comment]
Strangely enough, the problem only occurs in a few cases. I also use uBlock in Chrome, without problems. It seems to be a race condition in the execution order.

In a hurry I don't know how to fix this. I will keep an eye on this.

**Comment by FlawTECH** [comment]
Thank you for your reply.

I forgot to mention my extensions don't update automatically (I'm running ungoogled chromium) so yesterday, I figured I'd just update and see. Now, after updating to the latest uBlock Origin dev build (1.35.3b7) it doesn't seem to be happening anymore.

I tried refreshing about a dozen times and never had this bug again, so if it's a race condition it could be pretty rare. My advice to people having this same bug is to just check if they are running the latest version of uBlock and maybe even try using the latest dev build from their github.
